### DISCUSSION TITLE
Integrate the API with an LLM

### DISCUSSION THREAD

**Discussion by Jazzyb21** [Veterans-United-Code-For-America/farmersmarkets-for-all#44] [Ideas]
We should create a chat bot that integrates with open ai so users can get general information about SNAP, WIC, benefits for their state or zip code.  @Mechasparrow @michael-navazhylau-vu

**Comment by jasmyneboggs** [comment]
Prototype: https://github.com/Jazzyb21/chat-bot/blob/main/chatbot.py
